### PR TITLE
Introduce Vaibhav-Kaushik

### PR DESCRIPTION
**Pull request by vaibhav-99**

### CODE REVIEW

**Review by gaurav1999 — COMMENTED** [review]
Hi Vaibhav, 
Good to see you, now I will be putting down some problems in the file which you did wrong.
You need to rework on that and this will give you hands on learning for git.

File: vaibhav_kaushik.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Name: Vaibhav Kaushik
```
Comment (gaurav1999) [inline vaibhav_kaushik.md:2]: The content is in markdown code format, like 

```This is the contetn nnnnnnnnnnnnnnnnnnnnnnnnnnnnnnttt``` , see the work of priyanka 's PR.

**Comment by gaurav1999** [comment]
Also vaibhav you might want to see this note:- https://github.com/gaurav1999/LearnGit/pull/2#issuecomment-502387581

File: vaibhav_kaushik.md (hunk 1 above)
Comment (vaibhav-99) [reply vaibhav_kaushik.md:2]: I have made the required changes, to make the text in markdown code format.

**Comment by gaurav1999** [comment]
@vaibhav-99 Got your changes, now we need to learn how to improve the commit history. Here you have got 3 commits but we want it to be the last one only to keep a linear history.

**Comment by gaurav1999** [comment]
@vaibhav-99 , good work. Let's get it merged.

**Pull request merged**